src/main/java/com/commercetools/sync/products/helpers/PriceReferenceResolver.java

```diff
@@ -216,27 +215,29 @@ public PriceReferenceResolver(
   }
 
   /**
-   * Given a {@link PriceDraftBuilder} this method attempts to resolve the customer group reference
-   * to return a {@link CompletionStage} which contains the same instance of draft builder with the
-   * resolved customer group reference.
+   * Given a {@link PriceDraftBuilder} this method attempts to resolve the customer group resource
+   * identifier to return a {@link CompletionStage} which contains the same instance of draft
+   * builder with the resolved customer group resource identifier.
    *
-   * <p>Note: The key of the customer group reference taken from the value of the id field of the
-   * reference.
+   * <p>Note: The key of the customer group reference taken from the value of the key field of the
+   * resource identifier.
```

Comment:
Do we need this Note? We can remove it, Since we don't have confusion anymore using key value in id field. wdyt?

Reply:
https://github.com/commercetools/commercetools-sync-java/pull/676/commits/113f60e19a3b214eba1831d9d9ad73567d8ea9e2